distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -58,8 +59,6 @@ a {
   }
 
   li {
-    margin-bottom: 10px;
-
     &:last-child {
       margin-bottom: 0;
     }
```

Comment:
We can get rid of this now that the list items won't have a `margin-bottom` by default.